src/autogluon/bench/eval/benchmark_context/utils.py

```diff
@@ -1,11 +1,112 @@
+from __future__ import annotations
+
+from urllib.parse import urlparse
+
+import boto3
+
 from autogluon.common.loaders import load_s3
 from autogluon.common.utils import s3_utils
 
 
-def get_s3_paths(path_prefix: str, contains=None, suffix=None):
+def is_s3_url(path: str) -> bool:
+    """
+    Checks if path is a s3 uri.
+
+    Params:
+    -------
+    path: str
+        The path to check.
+
+    Returns:
+    --------
+    bool: whether the path is a s3 uri.
+    """
+    if (path[:2] == "s3") and ("://" in path[:6]):
+        return True
+    return False
```

Comment:
Where did this code come from? It looks like a copy paste of things in `autogluon.common`.

Reply:
Bench depends on `autogluon.common`, just import the helper functions instead of duplicating them.